react-components/src/components/Output/Output.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component } from 'react';
+
+type Item = {
```

Comment:
please, try to be more specific in naming to be more precisely. It's a little abstract.